templates/nvidia/index.html

```diff
@@ -235,7 +236,7 @@ <h3 class="p-heading--4">AI-Ready platforms</h3>
 	<div class="row p-strip u-no-padding--top u-hide--small">
```

Comment:
```suggestion
	<div class="row p-strip u-no-padding--top u-hide--small u-hide--medium">
```

Reply:
I think this one we left it on purpose, as it can be viewable in medium-sized screens.